Bottom: b39db434325a86339f02637152f390f078a17d8b
Top:    1b9be72a8706542c3b39af0e13c142c01379419c
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-06-21 16:28:26 +0800

Refresh of tuntap-convert-to-napi

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 34176cc..e635ae5 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -637,6 +637,7 @@ static int tun_poll(struct napi_struct *napi, int budget)
 	struct sk_buff *skb;
 	unsigned received = 0;
 
+	/* FIXME: use skb_queue_splice_tail_init */
 	while (received < budget &&
 	       (skb = skb_dequeue(&tfile->socket.sk->sk_write_queue))
 		!= NULL) {
